mlir_graphblas/engine.py

```diff
@@ -653,7 +795,7 @@ def _lower_types_to_strings(
 
     def _generate_mlir_string_for_multivalued_functions(
         self, mlir_functions: Iterable[mlir.astnodes.Function], passes: List[str]
-    ) -> Tuple[str, str]:
+    ) -> Tuple[str, str, str]:
```

Comment:
I think I did not update this correctly. This should be `Tuple[str, List[str], List[str]]`.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/pull/98/commits/e39f25858cb5bd6abf266bde7eb30f7cf8c67bae fixes this.